boards/arm/stm32g071b_disco/stm32g071b_disco.dts

```diff
@@ -144,11 +144,10 @@
 		status = "okay";
 		compatible = "ti,ina230";
 		reg = <0x40>;
-		config = <INA230_CONFIG(
-			INA230_OPER_MODE_BUS_VOLTAGE_CONT,
-			INA230_CONV_TIME_1100,
-			INA230_CONV_TIME_1100,
-			INA230_AVG_MODE_1)>;
+		adc-mode = "Bus and shunt voltage continuous";
```

Comment:
this should be "Bus voltage continuous" I think, have this https://github.com/zephyrproject-rtos/zephyr/pull/62050/ already btw

Reply:
ah, good

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/62050 has been merged.
Please rebase